src/Microsoft.SqlTools.Hosting/Utility/SqlClientEventListener.cs

```diff
@@ -41,7 +42,7 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
             {
                 if (payload != null)
                 {
-                    Logger.Verbose($"eventTID:{eventData.OSThreadId} {payload.ToString()}");
+                    Logger.Verbose($"eventTID:{Environment.CurrentManagedThreadId} {payload.ToString()}");
```

Comment:
I did this because the thread the listener is running on isn't necessarily the same one that the event was logged on. So doing it this way could end up with the wrong thread ID.

Can you at least have an IFDEF that will still use the OSThreadId in netstandard2.1+ builds?